src/core/lib/resource_quota/memory_quota.cc

```diff
@@ -50,9 +50,11 @@ const ReclaimerQueue::Index ReclaimerQueue::kInvalidIndex;
 void ReclaimerQueue::Insert(
     std::shared_ptr<EventEngineMemoryAllocatorImpl> allocator,
     ReclamationFunction reclaimer, Index* index) {
-  MutexLock lock(&mu_);
```

Comment:
nit and optional: you might want to use `ReleasableMutexLock` intead.

Reply:
definitely. didn't know we had that knob.